components/app-core/backend/portal_proxy.go

```diff
@@ -4,8 +4,8 @@ import (
 	"database/sql"
 	"time"
 
-	"github.com/gorilla/sessions"
 	"github.com/SUSE/stratos-ui/components/app-core/backend/repository/interfaces"
+	"github.com/gorilla/sessions"
```

Comment:
We should add "go fmt" in the build steps. That would reorder the files automatically.